### PR TITLE
Start Freiburg project page

### PR DESCRIPTION
**Pull request by bebatut**
I created a page to collect all the projects we are involved on.
@erasche, @bgruening you have probably more projects to add 😄 

@bgruening, @erxleben how should we manage the collaboration/data analysis projects?

### CODE REVIEW

File: freiburg/projects.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+The Freiburg Galaxy team is involved in the maintenance and new development of HiCExplorer, and particularly its integration into Galaxy: [https://hicexplorer.usegalaxy.eu/](https://hicexplorer.usegalaxy.eu/)
+
+## de.NBI and ELIXIR
+
+The Freiburg Galaxy team is part of [de.NBI](http://www.denbi.de/), the German Network for Bioinformatics Infrastructure, through the [RNA Bioinformatics Center](http://www.denbi.de/rbc). Via de.NBI, the Freiburg Galaxy team is also involved in [ELIXIR](https://www.elixir-europe.org), as technical coordinator for the the [German node](https://www.elixir-europe.org/about-us/who-we-are/nodes/germany).
+
+The Freiburg Galaxy team is also co-leading the [ELIXIR Galaxy Community
+](https://www.elixir-europe.org/communities/galaxy), particularly with the maintenance of European Galaxy instance: [usegalaxy.eu](https://usegalaxy.eu/).
+
+## Neuromac? Single-cell?
```
Comment (bgruening) [inline freiburg/projects.md]: I'm not sure we should add data-analysis projects, thats hard to keep track. We should probably use the SFB terms here ... if we want to mention those.

File: freiburg/projects.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+The Freiburg Galaxy team is highly involved in many projects of the Galaxy community:
+
+- Galaxy core development and maintenance
+- Tool integration into Galaxy and their maintenance through
+    - [Intergalactic Utilities Commission](https://galaxyproject.org/iuc/)
+    - [Björn's repository](https://github.com/bgruening/galaxytools)
+- [Galaxy Training Material](https://galaxyproject.github.io/training-material/), a collection of Galaxy-related tutorials and its infrastructure, developed and maintained by the community
+- [Galaxy Docker](https://github.com/bgruening/docker-galaxy-stable) to provide an easy distributable full-fledged Galaxy installation
+- Development and maintenance of different Galaxy flavours
+    - [RNA-Workbench](http://bgruening.github.io/galaxy-rna-workbench/)
```
Comment (bgruening) [inline freiburg/projects.md]: Do we want to include all flavors which we have?

File: freiburg/projects.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+The Freiburg Galaxy team is involved in the maintenance and new development of HiCExplorer, and particularly its integration into Galaxy: [https://hicexplorer.usegalaxy.eu/](https://hicexplorer.usegalaxy.eu/)
+
+## de.NBI and ELIXIR
+
+The Freiburg Galaxy team is part of [de.NBI](http://www.denbi.de/), the German Network for Bioinformatics Infrastructure, through the [RNA Bioinformatics Center](http://www.denbi.de/rbc). Via de.NBI, the Freiburg Galaxy team is also involved in [ELIXIR](https://www.elixir-europe.org), as technical coordinator for the the [German node](https://www.elixir-europe.org/about-us/who-we-are/nodes/germany).
+
+The Freiburg Galaxy team is also co-leading the [ELIXIR Galaxy Community
+](https://www.elixir-europe.org/communities/galaxy), particularly with the maintenance of European Galaxy instance: [usegalaxy.eu](https://usegalaxy.eu/).
+
```
Comment (bgruening) [inline freiburg/projects.md]: MIRACUM ?
COST action?

**Comment by bgruening** [comment]
Is that ok for a first version?

File: freiburg/projects.md (hunk 1 above)
Comment (erxleben) [reply freiburg/projects.md]: I agree to Björn

**Comment by bgruening** [comment]
Thanks B!

**Pull request merged**